### PR TITLE
docs(k8s) Redirect k8s changelog to KIC

### PR DESCRIPTION
**Pull request by lena-larionova**
Based on https://konghq.atlassian.net/browse/DOCS-1318.

SInce `kong-enterprise-k8s` is deprecated as of KE 2.1.x, removing changelog topics and redirecting to KIC changelog in Github.

### CODE REVIEW

File: app/enterprise/k8s-changelog.md [hunk 1]
```diff
@@ -1,126 +1,4 @@
 ---
 title: Kong Enterprise k8s Changelog
+redirect_to: https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md
 ---
-
```
Comment (hbagdi) [inline app/enterprise/k8s-changelog.md:4]: No, we wish to keep this page around. It just doesn't need to be linked to from anywhere.

Comment (lena-larionova) [reply app/enterprise/k8s-changelog.md:4]: Ack - I thought this was versioned and would stay in older pages. Fixing.

Comment (lena-larionova) [reply app/enterprise/k8s-changelog.md:4]: Adding a "Kong Enterprise k8s is deprecated" note to the top of this file. What date should I put for this? The KE 2.1.3.0 GA date?

Comment (hbagdi) [reply app/enterprise/k8s-changelog.md]: > What date should I put for this? The KE 2.1.3.0 GA date?

Yes

**Review by hbagdi — APPROVED** [review]

**Review by tlblessing — APPROVED** [review]

**Pull request merged**